docker/regtest/docker-compose.yml

```
@@ -1,6 +1,14 @@
 version: "3"
 
 services:
+  nginx:
+    image: nginx
+    volumes:
+      - ./dockerfile-deps/joinmarket/nginx:/etc/nginx/templates
```

```suggestion
      - ./dockerfile-deps/nginx:/etc/nginx/templates
```
Can we move the nginx dir from `dockerfile-deps/joinmarket/nginx` directory one level up into `dockerfile-deps/nginx` ?

---

Thanks, done in [d020b9b](https://github.com/joinmarket-webui/joinmarket-webui/pull/129/commits/d020b9b6f2085dae9bc723c64f6ea5530428c862).